components/Sidebar/SidebarEpisode/SidebarEpisode.styles.ts

```diff
@@ -35,15 +26,8 @@ export const sidebarEpisodeStyles = makeStyles((theme: Theme) =>
       textIndent: '-2000vw'
     },
     title: {
-      display: 'grid',
-      gridTemplateColumns: 'min-content 1fr',
-      gridGap: `${theme.spacing(1)}px`,
-      alignItems: 'center',
-      marginTop: theme.typography.pxToRem(theme.spacing(1))
-    },
-    playIcon: {
-      alignSelf: 'start',
-      fontSize: theme.typography.pxToRem(48)
+      marginTop: '0',
+      marginBottom: '0'
```

Comment:
Instead of using these two properties, you can use `marginBlock: 0`. Also, when setting a numeric value without units, don't use a string.

Reply:
Fixed!